features/utils/world.ts

```diff
@@ -16,13 +17,32 @@ import {
   getUserCredentialByUserKey,
   getUserCredentialByAppAndUserPermissions,
 } from "./credentials";
+import type { Writable, Readable } from "node:stream";
+
+type Response = {
+  stdout: Buffer;
+  stderr: Buffer;
+  status: number | null;
+};
 
 export class OurWorld extends World {
   public env: { [key: string]: string } = {};
   public replacements: Replacements = {};
+  private _childProcess?: ChildProcessByStdio<Writable, Readable, Readable>;
   private _workingDir?: string;
   private _credentials?: Credentials;
-  private _response?: SpawnSyncReturns<Buffer>;
+  private _response?: SpawnSyncReturns<Buffer> | Response;
```

Comment:
You can update execCliKintone to return `Response` so that we don't need to check whether the `_response` is `SpawnSyncReturn<Buffer>` or `Response`.

Reply:
@mshrtsr 

You mean to update the `execCliKintoneSync` to return `Response`, don't you?

Because, for `execCliKintone`, we have to return `ChildProcess`, so the later steps can be executed (type answer, press `Enter`)

Reply:
Oh, it's my mistake.
Yes, I meantd `execCliKintoneSync`.

Thank you for updating the PR!